### PR TITLE
Adding a Poll Logic to the Polls app

### LINKED ISSUE #79: Adding Poll logic to model
Adding methods to the poll model and adding tests for them as well.

### PR DESCRIPTION
**Pull request by Sendy459**
main changes:
Adding create_poll method with validation to the Poll model Adding show_suggestions method to the Poll model

And testing for those methods

### What this PR does?

This PR adds  logic to the Poll app
And tests for the methods that were Added

### Notes for reviewer

Added methods to the model.py
and new tests for those methods in test.py

### Related Issue

Resolves #79 

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [x] This PR does not require updating any documentation

### CODE REVIEW

**Review by Ofekmn — DISMISSED** [review]

**Review by Ofekmn — APPROVED** [review]

**Comment by rotem2022** [comment]
Great work overall! Just one small suggestion: there is a redundant space in the "Documentation" checkbox. It should be marked as [x] instead of [x ].

**Review by rotem2022 — APPROVED** [review]

File: poll/tests.py [hunk 1]
```diff
@@ -73,3 +80,56 @@ def test_poll_end_time_after_event_start_time(self, poll1):
...
+        assert err.value.message == Poll.invalid_poll_error
+
+    def test_create_poll_with_invalid_end_time(self, event1):
+        poll = Poll()
+        with pytest.raises(ValidationError):
+            poll.create_poll(
+                event=event1,
+                max_suggestions=5,
+                end_time=event1.start_time + timedelta(hours=1),
+            )
```
Comment (talko611) [inline poll/tests.py:99]:
```suggestion
    def test_create_poll_with_invalid_end_time(self, event1):
        poll = Poll()
        with pytest.raises(ValidationError) as err:
            poll.create_poll(
                event=event1,
                max_suggestions=5,
                end_time=event1.start_time + timedelta(hours=1),
            )
        assert err.value.message == Poll.invalid_poll_error
```

**Review by talko611 — CHANGES_REQUESTED** [review]
Add a suggestion to one of the tests

**Comment by Ofekmn** [comment]
Please rebase

**Review by Ofekmn — DISMISSED** [review]

**Review by rotem2022 — APPROVED** [review]

**Review by kobihk — CHANGES_REQUESTED** [review]

**Review by kobihk — APPROVED** [review]

**Pull request merged**